src/index.html

```diff
@@ -4,9 +4,49 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <h1 align="center">Calendar</h1>
+
+
+    <div class="
+      calendar
+      calendar--start-day_san
+      calendar--month-length_28"
```

Comment:
```suggestion
      calendar--san
      calendar--28"
```
you don't need start-day and month-length here

Reply:
also it's sunday, not sanday